packages/mc-html-template/src/process-headers.ts

```diff
@@ -28,24 +32,33 @@ const mergeDirectives = (...directives: TDirective[]) =>
       ),
     {}
   );
-const toHeaderString = (directives: TDirective = {}) =>
```

Comment:
This was a bit confusing to have it as a separate function, as it's only used for the CSP case.